articles/knsh14.re

```diff
@@ -0,0 +1,59 @@
+= 新しい GUI ライブラリ Gio で遊んでみよう！
+
+2019年7月に開催されたGopherCon 2019@<fn>{knsh14_gophercon_link}で新しいGoのGUIライブラリが発表されました。
+これまでGoの苦手な分野としてGUIを持ったアプリケーションが開発しづらいという点が挙げられていましたが、これを解決する可能性を秘めたライブラリです。
+
+
+//footnote[knsh14_gophercon_link][https://www.gophercon.com/]
+
+== Gio とは？
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "？". (ja-technical-writing/no-exclamation-question-mark)